nuxeo-drive-operations/src/main/java/org/nuxeo/drive/operations/NuxeoDriveGetRootsOperation.java

```diff
@@ -61,7 +62,7 @@ public DocumentModelList run() {
         boolean allRepositories = true;
         HttpServletRequest request = (HttpServletRequest) ctx.get("request");
         if (request != null) {
-            String respositoryName = request.getHeader("X-NXRepository");
+            String respositoryName = request.getHeader(RenderingContext.REPOSITORY_NAME_REQUEST_HEADER);
```

Comment:
Not really since the Drive client only sets the header and the attribute is only set by the REST API.
`RenderingContext.REPOSITORY_NAME_REQUEST_PARAMETER` is the name of the request parameter.